bedrock/firefox/templates/firefox/releases/nightly-feed.xml

```diff
@@ -14,9 +14,11 @@
   <link rel="alternate" type="text/html" href="{{ url('firefox.notes', channel='nightly')|absolute_url }}"/>
   <icon>{{ static('img/firefox/nightly/favicon.png')|absolute_url }}</icon>
   <updated>{{ notes[0].modified.isoformat() }}</updated>
+  <webfeeds:cover image="{{ static('img/firefox/nightly/page-image.png')|absolute_url }}" />
+  <webfeeds:icon>{{ static('img/firefox/nightly/favicon-196.png')|absolute_url }}</webfeeds:icon>
   {% for note in notes %}
   <entry>
-    <title>{{ note.note|markdown|striptags|truncate(100) }}</title>
+    <title type="html">{{ note.note|markdown|striptags|safe|truncate(100) }}</title>
```

Comment:
Doh, the PR gets merged before leaving comments 😅 Anyways, I tried some filter pattens here and though the `safe` was okay.